stable/atlantis/templates/statefulset.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "atlantis.fullname" . }}
+  labels:
+    app: {{ template "atlantis.name" . }}
+    chart: {{ template "atlantis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "atlantis.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "atlantis.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      securityContext:
+        fsGroup: 1000
+      volumes:
+      {{- range $name, $_ := .Values.serviceAccountSecrets }}
+      - name: {{ $name }}-volume
+        secret:
+          secretName: {{ $name }}
+      {{- end }}
+      {{- if .Values.gitconfig }}
+      - name: gitconfig-volume
+        secret:
+          secretName: {{ template "atlantis.name" . }}-gitconfig
```

Comment:
I believe these should be fullname.  If you were to install this into the same namespace twice this would collide and error

Reply:
Correct, and done.